packages/loopback/test/acceptance/routing/routing.acceptance.ts

```diff
@@ -90,6 +94,68 @@ describe('Routing', () => {
     expect(result).to.have.property('body', 'TestApp');
   });
 
+  it('creates a new child context for each request', async () => {
+    const app = givenAnApplication();
+    app.bind('flag').to('original');
+
+    const spec = givenOpenApiSpec()
+      .withOperationReturningString('put', '/flag', 'setFlag')
+      .withOperationReturningString('get', '/flag', 'getFlag')
+      .build();
+
+    @api(spec)
+    class FlagController {
+      constructor(@inject('ctx') private ctx: Context) {
```

Comment:
That's basically what I am doing here. Instead of comparing `ctx` references (which yield ugly assertion failures), I bind a value to the per-request context and then verify that the second request does not see the value set by the first request.